docs/infra/pull-request-environments.md

```diff
@@ -0,0 +1,33 @@
+# Pull request environments
+
+A temporary environment is created for each pull request that stays up while the pull request is open. Use cases for the temporary pull request environment includes:
+
+- Allow other delivery stakeholders—including product managers, designers, and business owners—to review changes before being merged and deployed
+- Enable automated end-to-end tests on the pull request
+- Enable automated accessibility checks on the pull request
```

Comment:
Some other ideas that come to mind:
- performance/load testing
- cross browser
- depending on how you define integration and e2e - maybe also various types of integration testing?
- Cross-browser/device testing (though this can overlap with a11y testing)
- Testing different env var or configurations

Reply:
great ideas!

load testing would probably need different environment configuration for cpu, num_tasks, and such rather than mirroring dev

security testing i think you could technically do with the existing stable environments like dev or staging

> depending on how you define integration and e2e - maybe also various types of integration testing?
> Cross-browser/device testing (though this can overlap with a11y and e2e testing)
> Testing different env var or configurations

definitely to all three